src/components/Card/Card.jsx

```diff
@@ -54,6 +55,11 @@ const CardWrapper = ({
   ...others
 }) => {
   const validOthers = filterValidAttributes(others);
+  const [isCardSelected, setIsCardSelected] = useState(false);
```

Comment:
does this actually need state? It looks like the prop should be able to handle it. See how `isResizable` is being used at line 251. I'd imagine this would work the same way

Reply:
Probably not. I originally thought we would have to pass state setters as arguments to onblur handlers when I first created this. I will double check and remove if unnecessary

Reply:
@sstone2423 removed! thanks